internal/proxy/configuration.go

```diff
@@ -0,0 +1,458 @@
+package proxy
+
+import (
+	"encoding/base64"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/micro/go-micro/config"
+	"github.com/micro/go-micro/config/source/env"
+	"github.com/mitchellh/mapstructure"
+	"golang.org/x/xerrors"
```

Comment:
My understanding is that we are using this simply to provide users with support for `Is`, `As`, `Unwrap` and `%w` in all 1.x go releases (whatever version of go they happen to be using). Is that correct?

Reply:
SSO is built from Go 1.14 now, so this is actually probably unnecessary. I believe this pull request was originally written prior to that update when we were on 1.12. Though I think I'd prefer in this case to keep it as is in this pull request, and return with a PR to refactor this in both places (`sso_auth` and `sso_proxy`) together. Let me know if you have any objections there!

Reply:
This pull request was originally written prior to SSO's update to Go 1.14, with it previously being on Go 1.12. I've moved this over to use the default `fmt` library (I think the main motivation here was to enable the use of `%w`, though now that's supported as of Go 1.13).

I'll look to update this also in `internal/auth/configuration.go` to realign the two - there are also a couple other changes to make).

Reply:
This pull request was originally written before SSO's move to Go 1.14 (previous to that it was on Go 1.12). I believe the main initial motivation was to enable use of `%w`, though now that's supported (as of Go 1.13) I think we can move to use the default `fmt` and `errors` libraries.

I'll look to open a subsequent pull request to make the same change in `sso_auth` to realign the two on this. (there are also a few other similar changes that need to be made)